### ISSUE TITLE
Add License options for submitted files

### ISSUE THREAD

**Issue by q3k** [joeferner/kicadcloud.com#21]
It would be great to have an option to specify under what license are the parts submitted - whether, for example, public domain (-like), MIT-like, or GPL-like.

This would allow to select between high-quality parts by [smisioto](http://smisioto.no-ip.org/elettronica/kicad/kicad-en.htm) which are under a GPL-like license and therefor unusable for some, and a public domain set of parts which could be lower quality but usable by everyone.

Additionally, it would be beneficial if the submitter had to sign off (like a git sign off somehow?) the parts they submit that they are known to be under such license and such copyright owner(s). Esepcially as I seem to see eagle imported parts in the current library, which have an unclear legal status.

**Comment by kasbah** [comment]
More of an aside but: footprints are not software. There aren't really any reasons why a GPL footprint wouldn't be usable by some. You can pretty much do whatever you want with it. I wish people picked a less confusing license for their parts.

Having the license visible with whatever you are downloading is a good idea however. This site is a bit outdated now (uses the legacy footprint format) but I do still find myself using it sometimes so would be really great to see some new features added (though I am not holding my breath).

**Comment by joeferner** [comment]
I agree with @kasbah footprints are not software so GPL doesn't really apply. I'm not even sure if legally it would hold up because software is so different.

@kasbah I've been keeping the site up because it doesn't really cost me anything and people do occasionally use it. I don't really have plans on updating it anytime soon mainly because I myself don't really use it. As KiCad has evolved so has my usage of it and I have come up with new workflows which handle versioning and management very differently.
